src/openrct2/localisation/Localisation.h

```diff
@@ -131,6 +131,16 @@ class Formatter
     {
         static_assert(sizeof(TSpecified) <= sizeof(uintptr_t), "Type too large");
         static_assert(sizeof(TDeduced) <= sizeof(uintptr_t), "Type too large");
+
+        // checking if the specified type is a valid one
+        constexpr bool validType = std::is_same<TSpecified, char*>::value || std::is_same<TSpecified, const char*>::value
```

Comment:
Since c++17 you can simplify this to std::is_same_v then you don't need the ::value